index.html

```diff
@@ -164,22 +204,47 @@ <h3 class="card-title">
         </div>
     </div>
         </section>
-        <footer class="contact-me display-flex flex-column align-middle">
+        <div class="separator-blank"></div>
+        <footer class="contact-me" id="contact">
             <h2 class="section-heading neutrals txt-center">
                 I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. 
             </h2>
-            <div class="separator-blank"></div>
-            <form class="my-contact-form display-flex flex-column align-middle"
-            action="https://formspree.io/f/xdojwqqv"
-            method="POST"
-            >
-                <input type="text" name="name" maxlength="30" placeholder="Full name" required>
-                <input type="email" name="email" placeholder="Email Adress" required>
-                <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
-                <button class="button btn-primary border-none" type="submit">Get In touch</button>
-            </form>
+            <div class="mobile-form">
+                <form class="my-contact-form"
+                action="https://formspree.io/f/xdojwqqv"
+                method="POST"
+                >
+                    <input type="text" name="name" maxlength="30" placeholder="Full name" required>
+                    <input type="email" name="email" placeholder="Email Adress" required>
+                    <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+                    <button class="button btn-primary border-none" type="submit">Get In touch</button>
+                </form>
+            </div>
+            <div class="desktop-form">
+                <form class="my-contact-form"
+                action="https://formspree.io/f/xdojwqqv"
+                method="POST"
+                >
+                    <fieldset>
+                    <input type="text" name="first-name" maxlength="30" placeholder="First name" required>
+                    <input type="text" name="last-name" maxlength="30" placeholder="Last name" required>
+                    </fieldset>
+                    <input type="email" name="email" placeholder="Email Adress" required>
+                    <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+                    <button class="button btn-primary border-none" type="submit">Get In touch</button>
+                </form>
```

Comment:
[OPTIONAL] Kindly notice that hovering over your form button doesn't change the cursor to pointer. If you want, you can add this effect:
```css
.button {
    cursor: pointer;
}
```